lang/cs/Org.Apache.REEF/Org.Apache.REEF.nuspec

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<package >
+  <metadata>
+    <id>Org.Apache.REEF</id>
```

Comment:
I don't think this NuGet should be called just `Org.Apache.REEF`. Maybe `Org.Apache.REEF.FatNuGet`? In java land, we call these things "fat" jars. But maybe there is a better name?